### ISSUE TITLE
What is the roadmap for accessing ink content on OneNote pages?

### ISSUE THREAD

**Issue by asampal** [microsoftgraph/microsoft-graph-docs#7877]
[Enter feedback here]
Making this request from the graph explorer:

`https://graph.microsoft.com/beta/me/onenote/pages/0-f184a990a12848f8acd94fd106799f14!1-68BE26BCCB89C739!796/content`

to retrieve the content from a OneNote page gave the following content:

```hmtl
<html lang="en-CA">
	<head>
		<title>123123￼</title>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
		<meta name="created" content="2020-04-22T13:17:00.0000000" />
	</head>
	<body data-absolute-enabled="true" style="font-family:Calibri;font-size:11pt">
		<!-- InkNode is not supported -->
	</body>
</html>
```

Is there any alternative to the graph API for accessing OneNote ink content? If not, what is the timeline for exposing ink nodes? I'm particularly interested in creating the ink outside of OneNote and inserting it into the page as well as grabbing the ink, editing it outside of OneNote, and possibly re-inserting it.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 94fd8e29-adcf-aa46-d4b2-a49d1e2d02f9
* Version Independent ID: d6899161-3e69-2198-f445-882dfd879003
* Content: [Get resource - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/resource-get?view=graph-rest-beta&tabs=http)
* Content Source: [api-reference/beta/api/resource-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/beta/api/resource-get.md)
* Product: **onenote**
* Technology: **microsoft-graph**
* GitHub Login: @jewan-microsoft
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by cjj19970505** [comment]
There is a [official blog](https://developer.microsoft.com/en-us/onenote/blogs/onenote-ink-beta-apis/) talked about this. But  this is already a blog back in 2017. Since then there are no more document talked about ink. The api mentioned in that blog still works on MS Graph v1.0 but didnt adress how to update ink data. So why these set of API are undocumented ? Will they get futher support or will be deprecated or become unavailable in the future ?

**Comment by asampal** [comment]
@cjj19970505 , yes that blog is where I initially read about this. I'm guessing that inking, at least exposing it to apps other than OneNote, is a dead end. At one point MS seemed very excited about the potential, but in recent years the whole thing seems to have died.

**Comment by cjj19970505** [comment]
> @cjj19970505 , yes that blog is where I initially read about this. I'm guessing that inking, at least exposing it to apps other than OneNote, is a dead end. At one point MS seemed very excited about the potential, but in recent years the whole thing seems to have died.
> 
> It's a shame because there could be some great integration done from hardware like the Onyx Boox large 10"/13" devices which have really nice inking capability and a "fast-enough" refresh, given the e-Ink screens.

that’s exactly what I am trying to do but found it is a dead end for now. I was trying to write a OneNote client for my onyx boox air. I am a heavy OneNote user but my eye just getting tired after long time staring at my surface pro. It will be very useful to be able to operate inking data from the ms graph apis

**Comment by asampal** [comment]
I think that with the undocumented/unfinished state of the API you could still get something done, i.e. grab the ink that was generated on the OneNote side, edit it on the Boox device and then insert a new top-level ink node into the OneNote document.

IIRC when I looked, despite the  in my OP, the ink data is there in the response and the that message only applied to the MS tool for exploring the data.
